training/exercises/02-Icebreaker.md

```diff
@@ -12,7 +12,7 @@ This is an exercise to walk through a simple contribution flow.
 Info to add:
 
 <pre>
-Hello my name is Aimee.
+Hello my name is Sri Harsha.
```

Comment:
This line is an example and should not be changed. You should only add your own introduction.